### DISCUSSION TITLE
compilation failing on helix-term

### DISCUSSION THREAD

**Discussion by Dorfeuheinz** [helix-editor/helix#7537] [Troubleshooting]
### Summary

```
 Stderr: error: Server does not allow request for unadvertised object bc26a6a949f2e0d98b7bfc437d459b250900a165

error: failed to compile `helix-term v0.6.0 (/home/user/helix/helix-term)`, intermediate artifacts can be found at `/home/user/helix/target`
```

Im recieving this error when im trying to build helix
Please fix this asap

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:

Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Linux (deepin)

### Terminal Emulator

Deepin terminal

### Helix Version

latest (as mentioned in docs)

**Comment by the-mikedavis** [comment]
What is your `git --version`?

**Reply by Dorfeuheinz** [reply]
2.20.1

**Reply by the-mikedavis** [reply]
You can either update your `git` or remove tree-sitter-hare from the build by creating a file `~/.config/helix/languages.toml` with the contents:

```toml
# ~/.config/helix/languages.toml
use-grammars = { except = ["hare"] }
```

See https://github.com/helix-editor/helix/issues/2535

**Reply by Dorfeuheinz** [reply]
creating a languages.toml isnt working, it shows a different kind of error. As far as updating git goes, what version do you want me on?

**Reply by Dorfeuheinz** [reply]
`
error: failed to run custom build command for `helix-term v0.6.0 (/home/user/helix/helix-term)`

Caused by:
  process didn't exit successfully: `/home/user/helix/target/release/build/helix-term-df24a2da7dd2a6c7/build-script-build` (exit status: 101)
error: failed to compile `helix-term v0.6.0 (/home/user/helix/helix-term)`, intermediate artifacts can be found at `/home/user/helix/target`

`
I tried deleting the "hare" portion i the language server & this is the result
Idk what to do

**Comment by Dorfeuheinz** [comment]
I did a few things & it finally works now. I could change the language.toml as per your suggestion & now it works. Thankyou so much really appreciate the help
